stacks/apps/apple-api-bridge.yml

```diff
@@ -0,0 +1,349 @@
+# stacks/apps/apple-api-bridge.yml
+# 100A
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+
+Description: A serverless API application to batch api requests to Apple
+
+Conditions:
+  IsProduction: !Equals [!Ref EnvironmentType, Production]
+
+Parameters:
+  EnvironmentType: { Type: String }
+  EnvironmentTypeAbbreviation: { Type: String }
+  RootStackName: { Type: String }
+  RootStackId: { Type: String }
+  CloudWatchAlarmTaggerServiceToken: { Type: String }
+  CloudWatchLogGroupTaggerServiceToken: { Type: String }
+  CodeS3Bucket: { Type: String }
+  CodeS3ObjectKey: { Type: AWS::SSM::Parameter::Value<String> }
+
+Resources:
+  HttpApi:
+    Type: AWS::Serverless::HttpApi
+    Properties:
+      StageName: !Ref EnvironmentType
```

Comment:
Including a stage name here is not necessary when using Serverless resources, and somewhat duplicative since the entire deployment is for a given environment type. The resulting URLs are simpler if it's excluded, so I would drop it.